backend/lib/til_web/router.ex

```diff
@@ -21,5 +21,10 @@ defmodule TilWeb.Router do
 
     pipe_through :authenticated
     resources "/posts", PostController, only: [:create, :update, :delete]
+
+    scope "/activities", Activities do
+      put "/like", LikeController, :like
```

Comment:
what do you think about creating nested resource for posts?
https://hexdocs.pm/phoenix/routing.html#nested-resources